data/users/model/src/commonMain/kotlin/dev/burnoo/demo/listapp/data/users/model/User.kt

```diff
@@ -0,0 +1,11 @@
+package dev.burnoo.demo.listapp.data.users.model
+
+data class User(
+    val title: String,
+    val firstName: String,
+    val lastName: String,
+    val photoUrl: String,
+    val gender: String,
+    val email: String,
+    val phone: String,
```

Comment:
Consider value objects for all these strings. It's easy to make a mistake and swap arguments when calling the constructor. :)